### PR TITLE
enable aes cfb mode

### PR DESCRIPTION
**Pull request by la-luo**
## Description of changes in release / Impact of release:
OpenPGP is using AES CFB mode and we are missing it in larky

## Documentation
(insert text here)

### Is this a breaking change?
- [ ] Yes
- [x] No

### If you answered Yes then describe why is it so
(insert text here if applicable)

### Is there a way to disable the change?
- [x] Use previous release
- [ ] Use a feature flag
- [ ] No

#### Additional details go here
(insert text here if applicable)

### CODE REVIEW

File: larky/src/main/resources/vendor/Crypto/Cipher/_mode_cfb.star
Comment (mahmoudimus) [inline larky/src/main/resources/vendor/Crypto/Cipher/_mode_cfb.star:211]: We have a built in divmod as well. This is not necessary.

File: larky/src/test/resources/vendor_tests/pycryptodome/Crypto/Cipher/test_CFB.star
Comment (mahmoudimus) [inline larky/src/test/resources/vendor_tests/pycryptodome/Crypto/Cipher/test_CFB.star:12]: You will need a lot more comprehensive test suite than this. There are usually many edge cases that we have to run through but I think this is a good first pass.

File: larky/src/main/resources/vendor/Crypto/Cipher/_mode_cfb.star
Comment (mahmoudimus) [inline larky/src/main/resources/vendor/Crypto/Cipher/_mode_cfb.star:263]: You need to add a `__name__` here as well

**Review by mahmoudimus — CHANGES_REQUESTED** [review]

File: larky/src/main/resources/vendor/Crypto/Cipher/__init__.star
Comment (la-luo) [reply larky/src/main/resources/vendor/Crypto/Cipher/__init__.star:42]: yes, otherwise we cannot invoke aes with this new cipher mode

File: larky/src/main/resources/vendor/Crypto/Cipher/_mode_cfb.star
Comment (la-luo) [reply larky/src/main/resources/vendor/Crypto/Cipher/_mode_cfb.star]: done

File: larky/src/main/resources/vendor/Crypto/Cipher/_mode_cfb.star
Comment (la-luo) [reply larky/src/main/resources/vendor/Crypto/Cipher/_mode_cfb.star]: removed

File: larky/src/test/resources/vendor_tests/pycryptodome/Crypto/Cipher/test_CFB.star
Comment (la-luo) [reply larky/src/test/resources/vendor_tests/pycryptodome/Crypto/Cipher/test_CFB.star]: added more tests

File: larky/src/main/resources/vendor/Crypto/Cipher/_mode_cfb.star
Comment (la-luo) [reply larky/src/main/resources/vendor/Crypto/Cipher/_mode_cfb.star]: done

File: larky/src/main/resources/vendor/Crypto/Cipher/__init__.star
Comment (mahmoudimus) [reply larky/src/main/resources/vendor/Crypto/Cipher/__init__.star:42]: Sorry, let me clarify. Do you think we need to also implement `_create_openpgp_cipher` as well?

File: larky/src/test/resources/vendor_tests/pycryptodome/Crypto/Cipher/test_CFB.star
Comment (mahmoudimus) [inline larky/src/test/resources/vendor_tests/pycryptodome/Crypto/Cipher/test_CFB.star:67]: Is this right?

File: larky/src/test/resources/vendor_tests/pycryptodome/Crypto/Cipher/test_CFB.star
Comment (mahmoudimus) [inline larky/src/test/resources/vendor_tests/pycryptodome/Crypto/Cipher/test_CFB.star:60]: FWIW, when I meant to add more tests, I meant you had to port over: https://github.com/Legrandin/pycryptodome/blob/master/lib/Crypto/SelfTest/Cipher/test_CFB.py as I did for the GCM, ECB, etc. It's fine for now if you're comfortable w/ this as a first pass. But I would say these tests are insufficient.

File: larky/src/test/resources/vendor_tests/pycryptodome/Crypto/Cipher/test_CFB.star
Comment (la-luo) [reply larky/src/test/resources/vendor_tests/pycryptodome/Crypto/Cipher/test_CFB.star:67]: yes, they all get passed

Comment (mahmoudimus) [reply larky/src/test/resources/vendor_tests/pycryptodome/Crypto/Cipher/test_CFB.star:67]: I think you made a mistake here, line 67 is not right. I don't think there's a `_testsuite` here that is added to that suite.

File: larky/src/test/resources/vendor_tests/pycryptodome/Crypto/Cipher/test_CFB.star
Comment (la-luo) [reply larky/src/test/resources/vendor_tests/pycryptodome/Crypto/Cipher/test_CFB.star:60]: got it... didn't realize that. Can we merge this pr for now for pgp?  I will add that test later

File: larky/src/test/resources/vendor_tests/pycryptodome/Crypto/Cipher/test_CFB.star
Comment (la-luo) [reply larky/src/test/resources/vendor_tests/pycryptodome/Crypto/Cipher/test_CFB.star:67]: ooops you are right let me remove it...

**Review by mahmoudimus — APPROVED** [review]

File: larky/src/test/resources/vendor_tests/pycryptodome/Crypto/Cipher/test_CFB.star
Comment (mahmoudimus) [reply larky/src/test/resources/vendor_tests/pycryptodome/Crypto/Cipher/test_CFB.star:60]: I approved changes, waiting on your tests to pass. Someone else needs to review so we can merge. @moehajj ?

**Review by moehajj — APPROVED** [review]

File: larky/src/test/resources/vendor_tests/pycryptodome/Crypto/Cipher/test_CFB.star
Comment (moehajj) [reply larky/src/test/resources/vendor_tests/pycryptodome/Crypto/Cipher/test_CFB.star:60]: 1 review should be good, just resolve these conversations once done because they will block the merge

File: larky/src/main/resources/vendor/Crypto/Cipher/__init__.star
Comment (la-luo) [reply larky/src/main/resources/vendor/Crypto/Cipher/__init__.star:42]: check pycrypto it is standard CFB not openpgp https://github.com/pycrypto/pycrypto/blob/7acba5f3a6ff10f1424c309d0d34d2b713233019/lib/Crypto/Cipher/blockalgo.py#L82
So far it looks like not necessary

**Pull request merged**
